packages/legend-graph/src/graph/metamodel/pure/packageableElements/domain/Property.ts

```diff
@@ -22,10 +22,12 @@ import type { AbstractProperty, PropertyOwner } from './AbstractProperty.js';
 import type { AnnotatedElement } from './AnnotatedElement.js';
 import type { TaggedValue } from './TaggedValue.js';
 import type { StereotypeReference } from './StereotypeReference.js';
+import { action, makeObservable, observable } from 'mobx';
 
 export class Property implements AbstractProperty, AnnotatedElement, Hashable {
   readonly _UUID = uuid();
   readonly _OWNER: PropertyOwner;
+  isBeingDragged = false;
```

Comment:
We never want to add state specific to the editor (UI) onto the metamodels. Also mobx dependency should not be added to the metamodels. We only add observables to the metamodels through the ObserverHelpers when loading it in the studio app for editing through the editors. 
If you need this property, it would go somewhere in ClassEditorState.